src/model/parsers/index.ts

```diff
@@ -51,6 +51,17 @@ export class DescriptionParser {
   }
 }
 
+export class ReferenceParser {
```

Comment:
Test for this?

Reply:
Oops, I'm v silly today.

Reply:
Definitely doesn't hurt anything, I would leave it in fo sho